src/v/redpanda/admin/api-doc/shadow_indexing.json

```diff
@@ -203,7 +203,22 @@
           "summary": "Invoke trimming on the local cache of tiered storage objects",
           "operationId": "cloud_storage_cache_trim",
           "nickname": "post_cloud_storage_cache_trim",
-          "parameters": []
+          "parameters": [
```

Comment:
Not sure if there is an opportunity to add a "description" field but I think that would be the right place to say that if the params aren't used, it trims it down to the (default) max cache size. We'll add that in the public version of the docs for sure.

Reply:
Happy to add if you provide me the text you'll add for the public docs. We can have it the same.

Reply:
@nvartolomei Sure! 

> If no parameters are included, this endpoint by default trims the cache to the maximum cache size. Use the 'bytes' and 'objects' parameters to specify the number of bytes and objects retained in the cache.